<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D Human Model Landmarks Labeling</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>260</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>选择指示(按照当前颜色指示点选)</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="left_neck">
               <property name="text">
                <string>1. 左颈点</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="right_neck">
               <property name="text">
                <string>2. 右颈点</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="front_neck">
               <property name="text">
                <string>3. 前颈点</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="left_acromion">
               <property name="text">
                <string>4. 左肩点</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="right_acromion">
               <property name="text">
                <string>5. 右肩点</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="left_armpit">
               <property name="text">
                <string>6. 左腋窝</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="right_armpit">
               <property name="text">
                <string>7. 右腋窝</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="left_elbow">
               <property name="text">
                <string>8. 左肘点</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="right_elbow">
               <property name="text">
                <string>9. 右肘点</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="left_wrist">
               <property name="text">
                <string>10. 左手腕(外侧)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="right_wrist">
               <property name="text">
                <string>11. 右手腕(外侧)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="left_nipple">
               <property name="text">
                <string>12. 左乳尖点</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="right_nipple">
               <property name="text">
                <string>13. 右乳尖点</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="6" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="high_waist">
               <property name="text">
                <string>14. 高腰点</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="waist">
               <property name="text">
                <string>15. 腰点</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="low_waist">
               <property name="text">
                <string>16. 低腰点</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="crotch">
             <property name="text">
              <string>17. 裆底点</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="left_knee">
               <property name="text">
                <string>18. 左膝</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="right_knee">
               <property name="text">
                <string>19. 右膝</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="9" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="left_ankle">
               <property name="text">
                <string>20. 左脚踝(内侧)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="right_ankle">
               <property name="text">
                <string>21. 右脚踝(内侧)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="10" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="left_hip">
               <property name="text">
                <string>22. 左臀</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="right_hip">
               <property name="text">
                <string>23. 右臀</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
